src/index.html

```diff
@@ -17,6 +17,93 @@
     />
   </head>
   <body>
+    <div class="menu">
+      <div class="item logo">
+        <a href="#">
+          <img
+            src="images/logo.png"
+            alt="Logo"
+          />
+        </a>
+      </div>
+
+      <nav>
+        <ul class="menu-items">
+          <li class="item">
+            <a
+              class="list is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="item item-3">
+            <a
+              class="list"
+              href="#"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="item item-4">
+            <a
+              class="list"
+              href="#"
```

Comment:
The `href="#"` is a placeholder. Consider replacing it with actual URLs or JavaScript functions to enable proper navigation.